docs/kaiatech/kaia-white-paper.md

```diff
@@ -445,7 +445,7 @@ The consensus process consists of the following three stages:
 
 #### Account Model and Smart Contract
 
-Kaia Blockchain offers scalability in service development through its expanded account model and smart contract capabilities. Smart contracts on the blockchain enhance the efficiency of transactions and contracts between individuals through contract automation, and the use of smart contracts has had a significant impact on the blockchain and dApp ecosystem. Contract conditions can be coded into smart contracts and automatically executed, solving the trusted intermediary issue. Smart contracts have allowed the blockchain ecosystem to create new business models and economic systems by reducing the cost and time required to complete transactions. Kaia Blockchain supports a KVM execution environment, which is designed to be fast and efficient, providing the best and swiftest development environment for dApp developers and projects.
+Kaia Blockchain offers scalability in service development through its expanded account model and smart contract capabilities. Smart contracts on the blockchain enhance the efficiency of transactions and contracts between individuals through contract automation, and the use of smart contracts has had a significant impact on the blockchain and dApp ecosystem. Contract conditions can be coded into smart contracts and automatically executed, solving the trusted intermediary issue. Smart contracts have allowed the blockchain ecosystem to create new business models and economic systems by reducing the cost and time required to complete transactions. Kaia Blockchain supports a DVM execution environment, which is designed to be fast and efficient, providing the best and swiftest development environment for dApp developers and projects.
```

Comment:
This PR changes DVM to KVM, but it is changing in opposite way. Can you revert back this?

Reply:
> This PR changes DVM to KVM, but it is changing in opposite way. Can you revert back this?

Rephrased